src/formatting/types.rs

```diff
@@ -933,7 +933,7 @@ fn join_bounds_inner(
                 _ => false,
             };
 
-            let shape = if i > 0 && need_indent && force_newline {
+            let shape = if need_indent && force_newline {
```

Comment:
The way the code is structured, `force_newline` will always be false on the first pass through and a second pass is only triggered in situations where we have to use multiple lines to format. As such this index check only serves to cause a bug